pkg/network/tcp_peer.go

```diff
@@ -226,10 +228,18 @@ func (p *TCPPeer) handleQueues() {
 func (p *TCPPeer) StartProtocol() {
 	var err error
 
+	var startHeight uint32
+	if p.isFullNode {
```

Comment:
Do we really need this flag? At least as a part of the `TCPPeer`? It might be useful locally here though, to _not_ start sync timer (this node won't have actual blocks anyway). Actuall the whole loop down below only makes sense for full node.

Reply:
Just removed this flag.